templates/financial-services/index.html

```diff
@@ -618,7 +618,7 @@ <h2>Increase developer productivity</h2>
       <p>An <a href="/desktop">Ubuntu workstation</a> is the <a href="https://recruit-c7ff.kxcdn.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf">professional developer’s preference</a>. Highly secure, reliable and with low cost of ownership, Ubuntu workstations are widely adopted by application engineering teams at financial institutions.</p>
       <p>Canonical provides 24x7 support and <a href="https://landscape.canonical.com/">management tools</a> for monitoring, managing, patching and compliance reporting on all your Ubuntu workstations.</p>
       <p>
-        <a href="https://buy.ubuntu.com/" >Learn about Ubuntu Advantage</a>
+        <a href="https://buy.ubuntu.com/" >Learn about Ubuntu Pro</a>
```

Comment:
```suggestion
        <a href="/pro" >Learn about Ubuntu Pro&nbsp;&rsaquo;</a>
```
Not sure if this matters really, since we have the redirect in place, but it's /pro on the doc (I believe the chevron should be there though)